src/stores/settings.ts

```diff
@@ -24,6 +24,12 @@ interface SettingsStore {
   setTheme: (theme: "dark" | "light") => unknown;
   vibrationEnabled: boolean;
   setVibrationEnabled: (vibrationEnabled: boolean) => unknown;
+  customRotatingCircleColor: boolean;
```

Comment:
Can we change this and all its similar references to `rotatingCircleColorEnabled`? Basically adding the `Enabled` prefix like for all the other toggles and dropping `custom`.